<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerMeterOptmizerWindow</class>
 <widget class="QMainWindow" name="PowerMeterOptmizerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Power Scan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <widget class="QGroupBox" name="control_groupBox">
      <property name="title">
       <string>Power_trace_on_off</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="11" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_power_max">
         <property name="decimals">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_power_min">
         <property name="decimals">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="interrupt_pushButton">
         <property name="text">
          <string>Interrupt</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Side power wheel min</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QDoubleSpinBox" name="side_powerwheel_min_doubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="start_pushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Side power wheel max</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="side_powerwheel_max_doubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="num_point_doubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Number of cycles</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="num_cycle_doubleSpinBox">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of points</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Openning time for turnning off</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="Dur">
         <property name="text">
          <string>Openning time for turnning on</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="time_turn_off_doubleSpinBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="time_turn_on_doubleSpinBox">
         <property name="suffix">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Min power(mW)</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Max power(mW)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="collect_groupBox">
         <property name="title">
          <string>Collect</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QCheckBox" name="collect_apd_checkBox">
            <property name="text">
             <string>APD</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="apd_int_time_doubleSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="2" rowspan="9">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_set_time">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>250</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>250</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Max. time to set</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>250</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>powerwheel step size</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_set_step_size">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>250</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_set_error">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>250</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>250</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>error</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="plot_groupBox">
      <property name="title">
       <string>Plot</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
